store/lib/store/taggable.ex

```diff
@@ -0,0 +1,232 @@
+defmodule Store.Taggable do
+  @moduledoc """
+  The Taggable context.
+  """
+
+  import Ecto.Query, warn: false
+
+  alias Store.Repo
+
+  alias Store.Taggable.Tag
+  alias Store.Taggable.Tagging
+
+  @doc """
+  Returns the list of tags.
+
+  ## Examples
+
+      iex> list_tags()
+      [%Tag{}, ...]
+
+  """
+  def list_tags do
+    Repo.all(Tag)
```

Comment:
Nah. Accept a limit as a parameter to `list_tag/1` at least. What if one has 1B tags?